### PR TITLE
Feature summary

### PR DESCRIPTION
**Pull request by hvyavuno**
Did not mean to delete this branch.

### CODE REVIEW

**Comment by DynataRidley** [comment]
Can you add a usage example in the readme file?

**Comment by DynataRidley** [comment]
Specifically under the "Supported API Functions" header like the other endpoints. I think it's OK that it doesn't have a link yet but it would be good to add the function there anyway.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ Links to the Demand API documentation are included for each function.
 [Get Project Detailed Report](https://developers.dynata.com/demand-api-reference/core-resources/projects/get-project-detailed-report): get_project_detailed_report(project_id)  
 [Get Pricing & Feasibility](https://developers.dynata.com/demand-api-reference/core-resources/pricing-feasibility/get-pricing-feasibility): get_feasibility(project_id)  
 [Get Invoice PDF](https://developers.dynata.com/demand-api-reference/billing_invoicing/invoicing/get-invoices): get_invoice(project_id)  
+[Get Invoices Summary PDF](https://developers.dynata.com): get_invoices_summary(\*\*kwargs)
```
Comment (dynata-bradley) [inline README.md]: I don't see documentation for this endpoint at that link?

Comment (wogsland) [reply README.md]: Test

Comment (hvyavuno) [reply README.md]: The documentation is not there. I have no knowledge of this.

Comment (hvyavuno) [reply README.md]: Rohan will take care of adding documentation once it is pushed out into production.

**Comment by dynata-bradley** [comment]
Since this was merged to `dev` I've opened issue #56 for the associated remaining technical debt.

**Pull request merged**